titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/KubeNotificationProcessor.java

```diff
@@ -182,7 +182,7 @@ public void shutdown() {
                 task.getId(),
                 current -> updateTaskStatus(podWrapper, newTaskStatus, executorDetailsOpt, node, current),
                 V3JobOperations.Trigger.Kube,
-                "Kube pod notification",
+                "Pod status updated from kubernetes node (k8s pod phase is now '" + event.getPod().getStatus().getPhase() + "')",
```

Comment:
`getStatus()` can be null. There are KubeUtils for safe access.

Reply:
earlier in the code it checks for `event.getPod().getStatus().getPhase() == null` on line 146, is that enough?

Reply:
If it also checks that `event.getPod().getStatus() != null` then yes.

Reply:
Yep. if that was null it would return early from line 146